rule.go

```diff
@@ -30,6 +30,9 @@ const (
 	delRuleHeaderType = netlink.HeaderType((unix.NFNL_SUBSYS_NFTABLES << 8) | unix.NFT_MSG_DELRULE)
 )
 
+// This constant is missing at unix.NFTA_RULE_POSITION_ID.
```

Comment:
Can you phrase this in the form of a TODO to include this constant in x/sys/unix please? (Or is there any reason why it wouldn’t be included there?)

Reply:
Done